### PR TITLE
[Repo] Upstream sync

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
Woohoo, we are [going to be] up to date with Wizden!

All the issues with this that I know of:
- [x] Maps won't load
- [x] Maps need to be resaved
- ~~Traitor codewords don't work~~ (Can't repro)
- ~~Flawed doesn't get any objective~~ (Can't repro)
- [x] Boxes are all invisible
- ~~Need to update emergency shuttle~~
- [x] Fix the client "ghost rsi" error
- [x] Fix spriteview
- [x] Fix announcements

### CODE REVIEW

File: Tools/gen_build_info.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 VERSION = os.environ['GITHUB_SHA']
 FORK_ID = "nyanotrasen"
-BUILD_URL = f"https://builds.nyanotrasen.moe/{{FORK_ID}}/builds/{{FORK_VERSION}}/{FILE}"
+BUILD_URL = f"https://cdn.nyanotrasen.moe/builds/{{FORK_VERSION}}/{FILE}"
 MANIFEST_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/manifest"
 MANIFEST_DOWNLOAD_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/download"
```
Comment (Pspritechologist) [inline Tools/gen_build_info.py:26]: Is this something we want to change? What do they do?

Comment (DEATHB4DEFEAT) [reply Tools/gen_build_info.py:26]: optional infrastructure we don't have

Comment (Pspritechologist) [reply Tools/gen_build_info.py:26]: We should probably tweak it just for the sake of not having their infrastructure in our repo. Comment it out, leave or blank, or whatever makes sense.

File: Resources/Server Info/RedialAddresses.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 # 3. Put comments on their own line
-# ss14://nyanotrasen.moe:1212
-# ss14://nyanotrasen.moe:1213
-# ss14://nfn.mooo.com:1213
+# Nyano
+ss14s://io.nyanotrasen.moe
+# Mining Station
+ss14://nfn.mooo.com:1213
+# Delta-V
+ss14://delta-v.org:1212
```
Comment (Pspritechologist) [inline Resources/Server Info/RedialAddresses.txt:9]: Huh! Neat.

Comment (Pspritechologist) [inline Resources/Server Info/RedialAddresses.txt]: Huh! Neat.

**Review by Pspritechologist — APPROVED** [review]
Mostly reviewing so if you push more in the future, I can compare exactly what it was :P
Take a look at the redial thing.

Also, until more gets pushed, feel free to merge this whenever.

**Pull request merged**